dmx-web.js

```diff
@@ -48,14 +48,11 @@ function DMXWeb() {
 			}
 		}
 	})
-	io.set('log level', 1)
```

Comment:
newer version of socket.io use the debug npm package, easy debugging via ``DEBUG=* ./dmx-web.js``